src/test/java/org/cyclonedx/BomJsonGeneratorTest.java

```diff
@@ -37,14 +39,21 @@ public void after() {
     public void schema12GenerationTest() throws Exception {
         Bom bom =  createCommonBom("/bom-1.2.xml");
         BomJsonGenerator generator = BomGeneratorFactory.createJson(Version.VERSION_12, bom);
-        generator.generate();
         Assert.assertTrue(generator instanceof BomJsonGenerator12);
         Assert.assertEquals(CycloneDxSchema.Version.VERSION_12, generator.getSchemaVersion());
         File file = writeToFile(generator.toJsonString());
         JsonParser parser = new JsonParser();
         Assert.assertTrue(parser.isValid(file, CycloneDxSchema.Version.VERSION_12, true));
     }
 
+    @Test
+    public void schema12JsonObjectGenerationTest() throws Exception {
```

Comment:
This is a pretty bare bones test at current time but I added it to make sure nothing get's hosed up! We can certainly build it out more!